frontend/app/@types/express-session.d.ts

```diff
@@ -17,6 +18,7 @@ declare module 'express-session' {
       state: string;
     };
     createdCases: Record<string, PersonSinCase>;
+    editingCase?: SinCaseDto;
```

Comment:
I don't know if this should go in the global session definition. This should be scoped to the flow that uses it.